pkg/codegen/tmpl/core_resource.tmpl

```diff
@@ -10,10 +12,12 @@ type K8sResource = kinds.GrafanaResource[Spec, Status]
 // NewResource creates a new instance of the resource with a given name (UID)
 func NewK8sResource(name string, s *Spec) K8sResource {
 	return K8sResource{
-		Kind:       "{{ .KindName }}",
-		APIVersion: "v{{ .Version }}-alpha",
-		Metadata: kinds.GrafanaResourceMetadata{
-			Name:        name,
+		TypeMeta: v1.TypeMeta{
```

Comment:
changing the template to use standard k8s types not our helpers (this will all be removed soon anyway!)